flake-module.nix

```diff
@@ -133,37 +134,49 @@ in
                   This is an internal option, not meant to be set by the user.
                 '';
               };
+
+              # Derived options
+
+              finalPackages = mkOption {
+                type = types.attrsOf raw;
+                readOnly = true;
```

Comment:
It looks like `readonly` has no effect, because I can still set `finalPackages.foo = ./.;` (even `finalPackages = ./.;`) in user flake.nix?

Or does it just silently ignore user values?